src/lucky_cli/process_runner.cr

```diff
@@ -31,6 +31,9 @@ module LuckyCli
     end
 
     private def run(process, args)
+      # When Ctrl+C is pressed, wait for the child process to handle the signal and finish - rather than us immediately exiting and leaving it to log shutting down messages from the background
```

Comment:
```suggestion
      # When Ctrl+C is pressed, wait for the child process to handle the signal and finish - 
      # rather than us immediately exiting and leaving it to log "shutting down" messages from the background
```